kube/kd/deployment.yaml

```diff
@@ -0,0 +1,199 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hocs-queue-tool
+  labels:
+    version: {{.VERSION}}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: hocs-queue-tool
+  template:
+    metadata:
+      labels:
+        name: hocs-queue-tool
+        role: hocs-backend
+        version: {{.VERSION}}
+    spec:
+      imagePullSecrets:
+        - name: registry-credentials
+
+      containers:
+        - name: hocs-queue-tool
+          image: quay.io/ukhomeofficedigital/hocs-queue-tool:{{.VERSION}}
+          securityContext:
+            runAsNonRoot: true
+            capabilities:
+              drop:
+                - SETUID
+                - SETGID
+          env:
+            - name: JAVA_OPTS
+              value: '-Xms512m -Xmx512m -Dhttps.proxyHost=hocs-outbound-proxy.{{.KUBE_NAMESPACE}}.svc.cluster.local -Dhttps.proxyPort=31290 -Dhttp.nonProxyHosts=*.{{.KUBE_NAMESPACE}}.svc.cluster.local'
```

Comment:
`-Dhttp.nonProxyHosts=*.{{.KUBE_NAMESPACE}}.svc.cluster.local`

Probably don't need this - can be removed in the next change.